docs(installation): use RFC5735 IPs for examples

rwoll opened the pull request:
Updates examples to use IPs from [TEST-NET-* blocks][rfc] designated for
example documentation instead of likely live (albeit innocuous) router IPs.

[rfc]: https://datatracker.ietf.org/doc/html/rfc5735

mxschmitt commented:
I unfortunately changed this part a few days ago, would be awesome if you could rebase, then we can land it.

rwoll committed:
docs(installation): use RFC5735 IPs for examples

Updates examples to use IPs from [TEST-NET-* blocks][rfc5735] designated for
example documentation instead of likely live (albeit innocuous) router IPs.

[rfc5735]: https://datatracker.ietf.org/doc/html/rfc5735

rwoll commented:
@mxschmitt rebased! 🎊
